src/App.tsx

```diff
@@ -1,18 +1,37 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
+import { Pagination } from './components/Pagination';
 import { getNumbers } from './utils';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42)
-  .map(n => `Item ${n}`);
-
 export const App: React.FC = () => {
+  const [itemsPerPage, setItemsPerPage] = useState(5);
+  const [currentPage, setCurrentPage] = useState(1);
+  const [total] = useState(42);
```

Comment:
no hardcoding pleazzzze ))

and if you're not using stateSetter - why even have a state, right?
regular variable would do.

```suggestion
  const total = currentItems.length;
```